### DISCUSSION TITLE
Composition of custom element ?

### DISCUSSION THREAD

**Discussion by revskill10** [kennyfrc/cami.js#2] [Q&A]
Do you have examples to illustrate composition of custom element inside html tag ?

**Comment by kennyfrc** [comment]
Hi @revskill10! Migrated this over to discussions. There are a few examples here: [https://github.com/kennyfrc/cami.js/tree/master/examples](https://github.com/kennyfrc/cami.js/tree/master/examples)

You can copy paste these in any HTML file and try it out.
